static/sass/_v1_pattern_lists.scss

```diff
@@ -119,3 +121,57 @@
     }
   }
 }
+
+@mixin ubuntu-p-is-trisected {
+
+  [class*='p-list'] {
+
+    &.is-trisected {
+
+      @media (min-width: $breakpoint-medium) {
+        display: flex;
+        flex-wrap: wrap;
+
+        .p-list__item {
+          width: calc(33.33% - .5rem);
+          margin-right: .5rem;
+
+          &:nth-child(3n+3) {
+            margin-right: 0;
+          }
+
+          &:nth-last-child(0) {
+            border-bottom: 0;
+          }
+        }
+      }
+    }
+  }
+}
+
+@mixin ubuntu-p-is-quartered {
+
+  [class*='p-list'] {
+
+    &.is-quartered {
+
+      @media (min-width: $breakpoint-medium) {
+        display: flex;
+        flex-wrap: wrap;
+
+        .p-list__item {
+          width: calc(25% - .5rem);
+          margin-right: .5rem;
+
+          &:nth-child(4n+4) {
+            margin-right: 0;
+          }
+
+          &:nth-last-child(0) {
+            border-bottom: 0;
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
it is a copy of the .is-divided pattern... perhaps if this is promoted we do that all at once?